Assets/Scripts/Room/LevelSelectManager.cs

```diff
@@ -50,6 +61,14 @@ public override void OnPlayerPropertiesUpdate(Player targetPlayer, Hashtable cha
             button.UpdateIndicators(levelNumber, isLocalPlayer);
         }
     }
+    public override void OnPlayerLeftRoom(Player otherPlayer)
+    {
+        base.OnPlayerLeftRoom(otherPlayer);
+
+        int levelNumber = (int)otherPlayer.CustomProperties[PLAYER_PROPERTIES_LEVEL_SELECTED];
```

Comment:
This line causes a `NullReferenceException` when, after a player has selected a level, the other player leaves _before_ selecting a level themselves. The exception occurs on the client of the player who remained in level select.